packages/cli/src/user-management/email/templates/passwordReset.mjml

```diff
@@ -0,0 +1,21 @@
+<mjml>
+	<mj-include path="./_common.mjml" />
+	<mj-body>
+		<mj-section>
+			<mj-column>
+				<mj-text font-size="24px" color="#ff6d5a">Reset your n8n password </mj-text>
+			</mj-column>
+		</mj-section>
+		<mj-section background-color="#FFFFFF" border="1px solid #ddd">
+			<mj-column>
+				<mj-text font-size="20px">Hi {{firstName}},</mj-text>
+				<mj-text>Somebody asked to reset your password on n8n ({{ domain }}). </mj-text>
```

Comment:
This might be out of scope, but should we add in a line to say "If you ignore this message, then your password will not be changed."? These emails often suggest contacting support if they didn't trigger the email reset too but perhaps we shouldn't do that?

Reply:
I think we should. but I'll ask ask the design folks.

@gandreini maybe you can help us phrase this?